server/autotest_server/testers/r/r_tester.py

```diff
@@ -19,11 +19,30 @@ def __init__(
 
         The result was created after running the tests in test_file.
         """
-        self._test_name = ":".join(info for info in [test_file, result.get("context"), result["test"]] if info)
+        # Format: [test_file] test_name
+        self._test_name = self._format_node_id(test_file, result)
         self.result = result["results"]
         super().__init__(tester)
         self.points_total = 0
 
+    @staticmethod
+    def _format_node_id(test_file: str, result: Dict) -> str:
+        """
+        Parse the test node id and format it as [relative_path] context test.
+
+        Original format: test_file::context::test or test_file::test
```

Comment:
The term "original format" is misleading here, and I was just remove it entirely. ("Original" is relative to older versions of the code, which any reader of the new code won't have direct access to.)